src/main/java/Event.java

```diff
@@ -1,4 +1,4 @@
-public class Event extends Task{
+public class Event extends Task {
 
     private String eventTime;

```

Comment:
Perhaps can consider removing the spaces ? Not sure if it violates the coding standards.